onnxruntime/test/optimizer/qdq_test_utils.h

```diff
@@ -363,30 +363,35 @@ GetQDQTestCaseFn BuildQDQSplitTestCase(
     const std::vector<int64_t>& input_shape,
```

Comment:
this will overwrite the previous test case. This doesn't look right. Why not create a new one